src/inmanta/server/services/resourceservice.py

```diff
@@ -906,6 +899,36 @@ async def update_fields_resource(
                         if not keep_increment_cache:
                             self.clear_env_cache(env)
 
+                        if status == ResourceState.failed or status == ResourceState.skipped:
```

Comment:
What made you change your mind about adding the new funcitonality here?

Reply:
iso6 compatiblity. If we only want to start restricting the functionality of resource_action_update in iso7, we need to support it in iso6.